app/static/src/app/components/mixins/selectVariables.ts

```diff
@@ -19,11 +19,13 @@ export default (
     const thisSrcGraphConfig = hasHiddenVariables ? "hidden" : graphIndex!.toString();
 
     const startDrag = (evt: DragEvent, variable: string) => {
-        const { dataTransfer } = evt;
+        const { dataTransfer, ctrlKey, metaKey } = evt;
```

Comment:
I dislike having to think to work out what abbreviations stand for - had to go back a line to decipher whether evt meant event or environment